src/server/server.ts

```diff
@@ -83,9 +88,19 @@ async function serveArticle(req: Request, res: ExpressResponse): Promise<void> {
             }
           },
           content => {
-            const { resources, element } = Page({ content, imageSalt, getAssetLocation });
+            const {
+              resources,
+              element,
+              hydrationProps
+            } = Page({ content, imageSalt, getAssetLocation });
+            const props = JSON.stringify({ ...hydrationProps, imageSalt })
+              .replace(/’/g, "&rsquo;")
+              .replace(/“/g, "&ldquo;")
+              .replace(/”/g, "&rdquo;")
+              .replace(/…/g, "&hellip;");
```

Comment:
Should this live in the transform function?

Reply:
Could be yes. All of these problems are introduced when I stringify the capi object for rehydration, will look for a way around it.

Reply:
It's not the stringify but it's node's `res.write` that is causing the problem...

Reply:
So this was all happening before `<meta charset="UTF-8"/>` was set 🤦‍♂️